app-utils/bili-sync/spec

```diff
@@ -0,0 +1,4 @@
+VER=2.1.1
+SRCS="git::commit=tags/v$VER::https://github.com/amtoaer/bili-sync"
+CHKSUMS="SKIP"
+CHKUPDATE="github::repo=amtoaer/bili-sync"
```

Comment:
Better create a project on [Anitya](https://release-monitoring.org) as it provides a unified update check method that we do not have to manually update the information ourselves (although sometimes we have to update the project on anitya) in case the upstream changed their platform.

Reply:
Unfortunately I am unable to login into Anitya, so I can't add this project now…